### ISSUE TITLE
e2e tests in github actions

### ISSUE THREAD

**Issue by Toumash** [Toumash/ruuster#30]
make it work https://github.com/Toumash/ruuster/actions/runs/9979529842

**Comment by Toumash** [comment]
Works fine? @theevilone45  https://github.com/Toumash/ruuster/actions/runs/9979656803/job/27579234065

**Comment by theevilone45** [comment]
Yes, I will add log "done" at the end of parent process, logs from  this job look correctly.
